apps/wallet-mobile/src/features/WalletManager/wallet-manager.ts

```diff
@@ -383,6 +385,38 @@ export class WalletManager {
     await this.#rootStorage.setItem('deletedWalletIds', [])
   }
 
+  getWalletPlate(publicKeyHex: string): WalletChecksum {
```

Comment:
`plate` is the text part of this checksum, I'd suggest `calcWalletChecksum`, this WalletChecksum type with TextPart and ImagePart let's hide it behind our own types, please return as `{plate, seed}` cuz the imagePart is actually the byte seeds. -> plate = TextPart, seed = ImagePart.

Reply:
let typescript infer the return for now

Reply:
even maybe only `checksum()`

Reply:
done